themes/bootstrap5/templates/layout/layout.phtml

```diff
@@ -12,7 +12,9 @@
     $this->layout()->searchbox = $this->context($this)->renderInContext('search/searchbox.phtml', []);
   }
 ?>
-<html lang="<?=$this->layout()->userLang?>"<?php if ($this->layout()->rtl): ?> dir="rtl"<?php endif; ?>>
+<html
+<?php /* data-bs-theme="dark" */ ?>
```

Comment:
Uncomment this line (for now) to simulate javascript-specified dark mode.  At this initial POC stage I'm more interested in how the code would actually support dark mode than how the user would indicate that they want it.

Reply:
In the meta tags (better) or in the CSS, I think we should set the color-scheme. This will cause some unspecified colors to follow the system, but it is a very easy way to implement user preference of color-scheme.

```html
<meta name="color-scheme" content="light dark">
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/meta/name/color-scheme

Reply:
I may be misreading that MDN page, but that meta tag seems to be for the document to say which color schemes it supports (and order of preference), rather than expressing the user's preference?  

That said, I'm fine using that meta tag to echo/reflect the user's preference (if they have expressed one via VuFind) -- just implemented that.  I don't see any immediate changes but as you say there may be some.

Reply:
Yes, but this will follow the system. User preference will have to be applied and stored separately - one of the current blind spots of modern web dev. If you change the meta tag to just "dark", it will apply a dark theme. You can then use the `prefers-color-scheme` and `light-dark` css tools.